modules/home-manager/zed-editor.nix

```
@@ -19,11 +24,11 @@ in
       userSettings.theme = {
         light =
           "Catppuccin "
-          + catppuccinLib.mkUpper cfg.flavor
+          + catppuccinLib.mkUpper cfg.lightFlavor
```

This would stay the same
```suggestion
          + catppuccinLib.mkUpper cfg.flavor
```